docs/source/getting-started/running-idom.rst

```diff
@@ -0,0 +1,294 @@
+Running IDOM
+============
+
+The simplest way to run IDOM is with the :func:`~idom.server.prefab.run` function. By
+default this will execute your application using one of the builtin server
+implementations whose dependencies have all been installed. Running a tiny "hello world"
+application just requires the following code:
+
+.. example:: hello_world
+    :activate-result:
+
+.. note::
+
+    Try clicking the **▶️ Result** tab to see what this displays!
+
+
+Running IDOM in Debug Mode
+--------------------------
+
+IDOM provides a debug mode that is turned off by default. This can be enabled when you
+run your application by setting the ``IDOM_DEBUG_MODE`` environment variable.
+
+.. tab-set::
+
+    .. tab-item:: Unix Shell
+
+        .. code-block::
+
+            export IDOM_DEBUG_MODE=1
+            python my_idom_app.py
+
+    .. tab-item:: Command Prompt
+
+        .. code-block:: text
+
+            set IDOM_DEBUG_MODE=1
+            python my_idom_app.py
+
+    .. tab-item:: PowerShell
+
+        .. code-block:: powershell
+
+            $env:IDOM_DEBUG_MODE = "1"
+            python my_idom_app.py
+
+.. dropdown:: Leave debug mode off in production!
```

Comment:
Writing this down before I forget - We should automatically enable `IDOM_DEBUG_MODE` when django_idom `settings.py:DEBUG = True`